build/petalinux.build

```diff
@@ -0,0 +1,3 @@
+export PETALINUX="/proj/petalinux/2020.1/petalinux-v2020.1_0227_1/tool/petalinux-v2020.1-final"
+export PETALINUX_VER=2020.1
```

Comment:
1. There is still some concerns about if we should have internal path in github.
2. This path points to a specific petalinux. How about if we want to use another build?
3. What is the purpose of this file? Should we let the script user setup the env? In this case, we only check env in build_edge.sh.